aim/aim_lib/nat_strategy.py

```diff
@@ -624,23 +633,25 @@ def _select_domains(self, objs, vmm_domains=None, phys_domains=None):
                 if phys_domains is not None:
                     obj.physical_domains = phys_domains
 
-    def _create_nat_epg(self, ctx, l3out, vmm_domains=None, phys_domains=None):
-        objs = self._get_nat_objects(ctx, l3out)
+    def _create_nat_epg(self, ctx, l3out, vmm_domains=None, phys_domains=None,
+                        epg_name=None):
+        objs = self._get_nat_objects(ctx, l3out, epg_name)
```

Comment:
This is a nit, but you probably should pass this as a named kwarg: epg_name=epg_name. That would guard against any changes in the future that might introduce more positional arguments.